docs/eventing/samples/ping-source/service.yaml

```diff
@@ -0,0 +1,11 @@
+# This is a very simple Knative Service that writes the input request to its log.
+
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+      - image: gcr.io/knative-releases/github.com/knative/eventing-contrib/cmd/event_display
```

Comment:
let's use a `gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display@sha256:.....` image ?

Reply:
latest is fine